Gemfile

```diff
@@ -139,7 +139,11 @@ platforms :ruby, :mswin, :mswin64, :mingw, :x64_mingw do
   gem "sqlite3", "~> 1.4"
 
   group :db do
-    gem "pg", "~> 1.1"
+    if RUBY_VERSION >= "3.2"
+      gem "pg", ">= 1.3.0.rc1"
```

Comment:
Can we always install 1.3.0.rc1?

Reply:
Thanks for the review.

I prefer to install `1.3.0.rc1` or higher only for Ruby 3.2.
There are two reasons. First, the current Gemfile setting allows pg 1.3.0 to be installed 
once it is released without any `rc` or other pre versions because it already installs `pg (1.2.3)`.

```
gem "pg", "~> 1.1"
```

Second, I assume it would bump the required versions of pg from 1.1 to 1.3, however, pg 1.3.0 is still an rc1 phase
and there is a blocking issue https://github.com/ged/ruby-pg/issues/398 to release pg gem 1.3.0. 

I have not found other strong reasons to bump the minimum versions of pg to 1.3 for users using Ruby 3.1 or lower versions of Ruby.

Reply:
The Gemfile is only used for development and test so changing it will not change the required version to use Rails. Conditionals inside Gemfile don't work great since they generate different Gemfile.lock output making development harder. I prefer to just remove those conditionals. Unless pg 1.3.0.rc1 doesn't work with Ruby 2.7 and 3.0.

Reply:
Sure. Let me update Gemfile.

Reply:
Updated and squashed. pg gem 1.3.0 should work with Ruby 2.7 and 3.0.

https://github.com/ged/ruby-pg/blame/master/History.rdoc#L60-L62

Removed:
- Remove support of ruby-2.2, 2.3 and 2.4. Minimum is ruby-2.5 now.
- Remove support for PostgreSQL-9.2. Minimum is PostgreSQL-9.3 now.